python/tchannel/event.py

```diff
@@ -40,8 +40,11 @@
     after_receive_request=0x21,
     before_receive_response=0x30,
     after_receive_response=0x31,
-    after_receive_error=0x40,
-    after_send_error=0x41,
+    after_receive_system_error=0x40,
+    after_receive_system_error_per_attempt=0x41,
```

Comment:
The naming here is confusing, what's the difference between `after_recieve` and `after_recieve_per_attempt`?